### PR TITLE
VACMS-4893: Fix spelling typo in word counseling.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #4893

## Testing done
Visual

## Implementation notes
**1. Create a PR that adds the new key** - current pr
2. Release it to prod
3. Edit the content on prod
4. Create a PR that removes the old key
5. Release to prod

 

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Review by kevwalsh — APPROVED** [review]
QA pass

File: config/sync/field.storage.taxonomy_term.field_vet_center_type_of_care.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ settings:
   allowed_values:
     -
       value: counselling
```
[the remark this answers is not in the document]
Comment (kevwalsh) [reply config/sync/field.storage.taxonomy_term.field_vet_center_type_of_care.yml:15]: @swirtSJW not sure if i understand the question. FWIW, this is all "new" data (vet center facility services), so it hasn't been pushed anywhere yet.

**Review by kevwalsh — DISMISSED** [review]
LGTM

File: config/sync/field.storage.taxonomy_term.field_vet_center_type_of_care.yml [hunk 2]
```diff
@@ -14,6 +14,9 @@ settings:
...
       label: 'Counselling services'
```
Comment (swirtSJW) [inline config/sync/field.storage.taxonomy_term.field_vet_center_type_of_care.yml:16]:
```suggestion
      label: 'deprecated counselling'
```

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
image

**Pull request merged**